_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,275 @@
+// ----------------------------
+// #MIXINS
+// Font Sizing
+// Line Height
+// Placeholder Styling
+// Support (Flex)
+// Headings (Mixin)
+//
+// #EXTENDS
+// Headings (Extend)
+// Polyfill (Flex)
+// Windows
+// Visual Disabled
+// Layering/Zindexing
+// Clear Children
+// Lists
+// Icon Fonts
+// Accessibility
+// Text Handling
+// Content Handling
+// ----------------------------
+
+
+// #MIXINS
```

Comment:
This formatting does not sync up with how we organize sections in the styleguide - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#table-of-contents. Please correct.